bubble-wrap.yaml

```diff
@@ -959,7 +958,7 @@ styles:
 
 scene:
     background:
-        color: global.scene1
+        color: [0.870,0.870,0.870]
```

Comment:
Looks like we should update the `earth1` global to `[0.870,0.870,0.870]` and then reference that global in the background section above.